packages/core/src/lib/interfaces.ts

```diff
@@ -1,7 +1,15 @@
-import { Breadcrumb, Context, SentryEvent } from './domain';
+import { Breadcrumb, Context, Scope, SentryEvent } from './domain';
 import { DSN } from './dsn';
 import { SendStatus } from './status';
 
+/**
+ * TODO
+ */
+export interface Scope {
```

Comment:
This is a duplicate of the export in `domain`. However, it actually belongs here and not in `domain`. 
Also, TODO.